src/agoradatatools/etl/utils.py

```diff
@@ -17,7 +17,9 @@ def _login_to_synapse(token: str = None) -> synapseclient.Synapse:
     Returns:
         synapseclient.Synapse: authenticated Synapse client session
     """
-    syn = synapseclient.Synapse()
+    version_number = synapseclient.__version__
+    agent_str = f"agora-data-tools/{version_number}"
```

Comment:
You won't want to add the version number of the synapseclient here. This version number is what the version of Agora data tools is.


There is already a user agent header that is automatically added that tells us what the version of the Synapse Client is

Reply:
Thank you! Sorry I misunderstood! I swtiched back to use 0.0 now